var/spack/repos/builtin/packages/rsem/package.py

```diff
@@ -18,15 +18,16 @@ class Rsem(MakefilePackage):
     version("1.3.1", sha256="93c749a03ac16e94b1aab94d032d4fd5687d3261316ce943ecb89d3ae3ec2e11")
     version("1.3.0", sha256="ecfbb79c23973e1c4134f05201f4bd89b0caf0ce4ae1ffd7c4ddc329ed4e05d2")
 
-    depends_on("c", type="build")  # generated
-    depends_on("cxx", type="build")  # generated
-
+    depends_on("c", type="build")
+    depends_on("cxx", type="build")
     depends_on("r", type=("build", "run"))
     depends_on("perl", type=("build", "run"))
     depends_on("python", type=("build", "run"))
     depends_on("bowtie")
     depends_on("bowtie2")
     depends_on("star")
+    # The bundled version of samtools/htslib is not compatible with zlib-ng
```

Comment:
Can you expand on this? There should be a compatibility mode for `zlib-ng`

Reply:
It doesn't work even with the compatibility mode enabled. It's some issue with the block size. See this discussion: https://github.com/samtools/htslib/issues/1257

Reply:
Hmm, the last comment says:

> Still, it's much better than native zlib, so I hope in time zlib-ng will simply become the standard system zlib given the compatible API.

and the issue seems fixed in https://github.com/samtools/htslib/pull/1258

Reply:
Is it possible to _not_ rely on vendored `samtools`?

Reply:
It's not supported in the Makefile, but I suppose it could be modified to use an external version. Looking at the history of the Makefile, it has relied on samtools-1.3/htslib-1.3 for ten years now, and rsem overall has actually not been updated in three years. So I would be reluctant to change the version here.

Reply:
Fair enough, thanks for the explanation. If you want to look into removing this limitation in the recipe, that would be welcome, but it can be in another PR.